doc/rados/operations/stretch-mode.rst

```diff
@@ -157,28 +157,27 @@ possible, if needed).
 
 .. _Changing Monitor elections: ../change-mon-elections
 
-Stretch Mode Limitations
-========================
-As implied by the setup, stretch mode only handles 2 sites with OSDs.
+Limitations of Stretch Mode 
+===========================
+Stretch mode can be used only with two sites using OSDs.
 
-While it is not enforced, you should run 2 monitors in each site plus
-a tiebreaker, for a total of 5. This is because OSDs can only connect
-to monitors in their own site when in stretch mode.
+Two monitors plus a tiebreaker monitor should be run in each data center, for a
+total of five monitors. While in stretch mode, OSDs can connect only to monitors
+in the data center of which they themselves are a part.
```

Comment:
I think the original was slightly ambiguous, leading to a semantic change in your revision.  @gregsfortytwo should validate my suggested rephrasing, in case I have it all wrong:

Two monitors should be run in each data center, plus a tiebreaker in a third (or the Cloud) for a
total of five monitors. While in stretch mode, OSDs will connect only to the tiebreaker and to monitors within the data center in which they are located.

Reply:
I endorse this. @gregsfortytwo: Check it out.

Reply:
I'm committing Anthony's revised text now.

Reply:
This is wrong, OSDs explicitly *do not* connect to the tiebreaker monitor. They will only connect to the (two, for this reason) monitors in their data center.

Reply:
@gregsfortytwo Thanks for clearing this up. I've altered the text, and I offer it to you again. Let us know if it's accurate now.